Network/src/main/kotlin/com/infomaniak/core/network/networking/UrlTraceInterceptor.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Infomaniak Core - Android
+ * Copyright (C) 2025 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.core.network.networking
+
+import io.sentry.Breadcrumb
+import io.sentry.Sentry
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class UrlTraceInterceptor : Interceptor {
+
+    override fun intercept(chain: Interceptor.Chain): Response {
+        val request = chain.request()
+        val response = chain.proceed(request)
+
+        response.header("x-request-id")?.let { requestId ->
+            val httpBreadcrumb = Breadcrumb.http(request.url.toString(), requestId, response.code)
```

Comment:
Is this http breadcrumb also triggered naturally somewhere else somehow? Because right now in sentry we already see some http breadcrumbs. Is the other one still needed? Is the other one not redundant with this one? Maybe both could be merged into a single one?